### PR TITLE
[V3 Cleanup] Cleanup users who have left the Guild

### PR DESCRIPTION
**Pull request by skmendez**
### Type

- [ ] Bugfix
- [ ] Enhancement
- [X] New feature

### Description of the changes
Implements #1545
Adds `cleanup former`, which removes a certain number of messages created by former Guild Members.

### CODE REVIEW

File: redbot/cogs/cleanup/cleanup.py [hunk 1]
```diff
@@ -237,6 +233,57 @@ def check(m):
...
+            prefixes.remove('')
+
+        def check(m):
+            for member in members:
+                if isinstance(member, discord.Member) and m.author == member:
+                    return False
+                elif m.author.id == member:  # Allow finding messages based on an ID
+                    return False
+            else:
+                return True
```
Comment (mikeshardmind) [inline redbot/cogs/cleanup/cleanup.py:270]: the `isinstance` here is unneccessary, as is the elif
.. actually the whole check function could be 
```py
def check(m):
    return not any(m.author == member for member in guild.members)
```

File: redbot/cogs/cleanup/cleanup.py [hunk 2]
```diff
@@ -237,6 +233,57 @@ def check(m):
...
+        author = ctx.message.author
+        is_bot = self.bot.user.bot
+        members = ctx.message.guild.members
+
+        if number > 100:
+            cont = await self.check_100_plus(ctx, number)
+            if not cont:
+                return
+
+        prefixes = await self.bot.get_prefix(ctx.message)  # This returns all server prefixes
```
Comment (mikeshardmind) [inline redbot/cogs/cleanup/cleanup.py:255]: This section does nothing... you don't use the prefixes at all

**Review by mikeshardmind — COMMENTED** [review]
There's a lot of dead code in this...

**Comment by Kowlin** [comment]
What I meant with #1545 was a addition to ``cleanup user`` to cleanup said user who has left the server.

**Comment by skmendez** [comment]
Closed due to misinterpretation of issue

**Pull request closed without merging**